src/main/java/ch/heigvd/dai/commands/Root.java

```diff
@@ -23,6 +49,10 @@ public void setPort(int port) {
     PORT = port;
   }
 
+  /**
+   * Get the port.
+   * @return the port.
+   */
   public static int getPort() {
```

Comment:
Maybe we can remove the static port and use:
https://github.com/heig-vd-dai-course/heig-vd-dai-course-java-ios-practical-content-template/blob/778e1934a64f338e93613afbb31dd9e92356d7c4/src/main/java/ch/heigvd/dai/commands/Read.java#L13